src/windows/line.rs

```diff
@@ -0,0 +1,150 @@
+// Licensed under the Apache License, Version 2.0, <LICENSE-APACHE or
+// http://apache.org/licenses/LICENSE-2.0> or the MIT license <LICENSE-MIT or
+// http://opensource.org/licenses/MIT>, at your option. This file may not be
+// copied, modified, or distributed except according to those terms.
+
+use pdb::{AddressMap, PdbInternalRva};
+use std::fmt::{Display, Formatter};
+
+#[derive(Clone, Debug)]
+pub struct Line {
+    pub rva: u32,
+    pub num: u32,
+    pub len: u32,
+    pub file_id: u32,
+}
+
+#[derive(Debug, Default)]
+pub struct Lines {
```

Comment:
Where is `Lines` used?  Am I missing something?

Reply:
Lines is used in pdb.rs.
It's a struct containing line data for each function.